Source/Renderer/Context.js

```diff
@@ -274,7 +274,15 @@ define([
         this._depthTexture = !!getExtension(gl, ['WEBGL_depth_texture', 'WEBKIT_WEBGL_depth_texture']);
         this._textureFloat = !!getExtension(gl, ['OES_texture_float']);
         this._fragDepth = !!getExtension(gl, ['EXT_frag_depth']);
+
+        var webgl1TimerQuery = getExtension(gl, ['EXT_disjoint_timer_query']);
+        var webgl2TimerQuery = getExtension(gl, ['EXT_disjoint_timer_query_webgl2']);
+        if (!!webgl1TimerQuery && !webgl2TimerQuery) {
+            webgl2TimerQuery = webgl1TimerQuery;
+        }
+
         this._debugShaders = getExtension(gl, ['WEBGL_debug_shaders']);
+        this._timerQuery = !!webgl1TimerQuery || !!webgl2TimerQuery;
```

Comment:
I think it is cleaner to group all the timer query code together, right?

Put this line above the `this._debugShaders = ...` line, and then put `this._disjointTimerQuery = ...` below this.